charts/akeyless-api-gateway/templates/deployment.yaml

```diff
@@ -148,6 +148,34 @@ spec:
             - name: CONFIG_PROTECTION_KEY_NAME
               value: {{ .Values.akeylessUserAuth.configProtectionKeyName }}
           {{- end }}
+          {{- if eq (include "akeyless-api-gw.adminAccessUidExist" .) "true"  }}
+            - name: ADMIN_UID_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "akeyless-api-gw.secretName" . }}
+                  key: admin-uid-init-token
+            - name:  GATEWAY_CLUSTER_CACHE
+              value: enable
+            - name: REDIS_ADDR
+              value: {{  include "akeyless-api-gw.fullname" . }}-cache-svc:6379
+            - name: REDIS_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "akeyless-api-gw.cacheSecretName" . }}
+                  key: cache-pass
+          {{- end }}
+          {{- if .Values.universalIdentity.uidRotationInterval }}
```

Comment:
If universal identitiy is depandent in "caching.enabled" then we should add a "required" statement for that. Also mention it in comment in the values.yaml file and readme

Reply:
caching.enabled ?  there is no filed named caching.enabled...
 redis is internal caching and the customer doesn't have any interaction with it or any say about it, 
there is no filed in values.yaml to control the redis,

Reply:
Check api-gw values.yaml file